backend/src/main/java/com/jobwait/persistence/answerpersistence/StringAnswer.java

```diff
@@ -0,0 +1,52 @@
+package com.jobwait.persistence.answerpersistence;
+
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.Types;
+
+import com.jobwait.domain.Answer;
+
+public class StringAnswer implements Answer<String> {
+    private String value;
+    private final int sqlType = Types.VARCHAR;
+    private String type;
+
+    public StringAnswer() {
+
+    }
+
+    public StringAnswer(String type, String value) {
+        this.type = type;
+        this.value = value;
+    }
```

Comment:
why two constructors?

Reply:
you need an empty one for Jackson, just more dogshit java boilerplate nonsense